assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/config.py

```diff
@@ -24,6 +24,9 @@ class SupportedTasks(_CustomEnum):
     ZERO_SHOT_IMAGE_CLASSIFICATION = "zero-shot-image-classification"
     EMBEDDINGS = "embeddings"
 
+    # BLIP2 task
+    IMAGE_CAPTIONING = "image-captioning"
```

Comment:
Same comment as the other place:

Should we instead use `image-to-text` to align with HF? 
https://huggingface.co/models?pipeline_tag=image-to-text